cfgov/v1/models/browse_page.py

```diff
@@ -141,7 +141,6 @@ class BrowsePage(AbstractBrowsePage):
             ("data_snapshot", organisms.DataSnapshot()),
             ("job_listing_table", JobListingTable()),
             ("yes_checklist", YESChecklist()),
-            ("raf_tool", v1_blocks.RAFTBlock()),
```

Comment:
The Python linter is failing because of the `from v1 import blocks as v1_blocks` line above; that needs to be removed or the linter complains about an unused import.